sources/_sector/tokens/balances_sources.yml

```diff
@@ -135,3 +135,18 @@ sources:
           - name: token_id
           - name: amount
           - name: type
+
+  - name: tokens_zksync
+    description: "Prices tables across blockchains"
+    tables:
+      - name: balances_zksync
```

Comment:
Hi @jeff-dude, do you know how to add `tokens_*.balances_*` for new chains?

Reply:
@lgingerich can you remove source file changes too?